staging/src/k8s.io/pod-security-admission/test/fixtures_seccomp.go

```diff
@@ -0,0 +1,137 @@
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompPod(p, seccompProfileLocalhost("testing"))
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileRuntimeDefault)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileLocalhost("testing"))
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompInitContainers(p, seccompProfileRuntimeDefault)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompInitContainers(p, seccompProfileLocalhost("testing"))
+				}),
+			}
+		},
+		generateFail: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureSeccompProfile(p)
+			return []*corev1.Pod{
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompPod(p, seccompProfileUnconfined)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompContainers(p, seccompProfileUnconfined)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompInitContainers(p, seccompProfileUnconfined)
+				}),
+			}
+		},
+	}
+
+	fixtureData_restricted := fixtureGenerator{
+		expectErrorSubstring: "seccomp profile",
+		generatePass: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureSeccompProfile(p)
+			return []*corev1.Pod{
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompPod(p, seccompProfileRuntimeDefault)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompPod(p, seccompProfileLocalhost("testing"))
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileRuntimeDefault)
+					setSeccompInitContainers(p, seccompProfileRuntimeDefault)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileLocalhost("testing"))
+					setSeccompInitContainers(p, seccompProfileLocalhost("testing"))
+				}),
+			}
+		},
+		generateFail: func(p *corev1.Pod) []*corev1.Pod {
+			p = ensureSeccompProfile(p)
+			return []*corev1.Pod{
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					setSeccompPod(p, seccompProfileUnconfined)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileRuntimeDefault)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompInitContainers(p, seccompProfileRuntimeDefault)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileRuntimeDefault)
+					setSeccompInitContainers(p, seccompProfileUnconfined)
+				}),
+				tweak(p, func(p *corev1.Pod) {
+					unsetSeccomp(p)
+					setSeccompContainers(p, seccompProfileUnconfined)
+					setSeccompInitContainers(p, seccompProfileRuntimeDefault)
+				}),
+			}
+		},
+	}
+
+	newFixtureRegistration("seccomp").
+		baseline(api.MajorMinorVersion(1, 0), fixtureData_baseline).
+		restricted(api.MajorMinorVersion(1, 0), fixtureData_restricted)
```

Comment:
I expected 1.0 fixtures registered for both which set annotations, and 1.19 fixtures registered for both which set fields